docs/CHANGELOG.md

```diff
@@ -2,10 +2,19 @@
 
 ### next
 
+  * tpm2_loadexternal:
+      - Added support to tpm2_loadexternal for parsing and loading the public
+        portion of a TSS2 Privkey PEM file.
```

Comment:
Can add a brief note on how to specify the pem file? You did this in tpm2_print where you mention the file must be specified as an argument.